site/docs/reference/Connectors/capture-connectors/PostgreSQL.md

```diff
@@ -0,0 +1,113 @@
+
+### Values
+| Value | Name | Type | Required/Default | Details |
+|-------|------|------|---------| --------|
+|  |Tenant |String| Required | The tenant in which to create the capture. This is the prefix of your cloud storage bucket. |
+| |Name | String | Required |The unique name of the capture |
+| `connectionURI` | Database Connection URL | String | Required | Connection parameters, as a libpq-compatible connection string |
+| `max_lifespan_seconds` | Maximum Connector Lifespan (seconds) | number | 0.0 | When nonzero, imposes a maximum runtime after which to unconditionally shut down |
+| `poll_timeout_seconds` | Poll Timeout (seconds) | number | 0.0 | When tail=false, controls how long to sit idle before shutting down |
+| `publication_name` | Publication Name | string | `"flow_publication"` | The name of the PostgreSQL publication to replicate from |
+| `slot_name` | Replication Slot Name | string | `"flow_slot"` | The name of the PostgreSQL replication slot to replicate from |
+| `watermarks_table` | Watermarks Table | string | `"public.flow_watermarks"` | The name of the table used for watermark writes during backfills |
+
+### Sample
+A minimal capture definition within the catalog spec will look like the following:
+
+```yaml
+captures:
+  $tenant/CAPTURE_NAME:
+    endpoint:
+      airbyteSource:
+        image: "ghcr.io/estuary/source-postgres:dev"
+        config:
+          connectionURI: "postgres://flow:flow@localhost:5432/flow"
+          # slot_name: “flow_slot”                     # Default
+          # publication_name: “flow_publication”       # Default
+          # watermarks_table: “public.flow_watermarks” # Default
+    bindings:
+      - resource:
+        stream: ${TABLE_NAME}
+        syncMode: incremental
+      target: ${COLLECTION_NAME}
+```
+We recommend using `flowctl discover` to generate it, as detailed below. Your capture definition will likely be more complex, with a  **binding** for each table in the source database.
+
+
+## How to use `flowctl discover`
+Currently, `flowctl discover` is the provided method to begin setting up a capture, and saves significant time
+compared to manually writing the catalog spec. `discover` generates the capture specification as well as the
+**collections** you'll need to perpetuate each bound resource within the Flow runtime.
+
+1. In your terminal, run:
+```console
+flowctl discover --image=ghcr.io/estuary/source-postgres:dev
+```
+This generates a config from the latest version of the connector, provided as a Docker image.
+
+2. Open the config file called `discover-source-postgres-config.yaml`. This is your space to specify the required values detailed above. Fill in `connectionURI` (required) and modify other values, if you'd like.
```

Comment:
Remove the mention of `connectionURI`, maybe just "fill in required values" for brevity here, since the generated placeholder config includes notes about which are required?